include/oneapi/mkl/dft/detail/types_impl.hpp

```diff
@@ -100,6 +101,8 @@ class dft_values {
     using real_t = std::conditional_t<prec == precision::SINGLE, float, double>;
 
 public:
+    std::set<config_param> changed;
```

Comment:
Do we really need to remember which param was changed? I thought we would assume anything can change between 2 commits.

Reply:
removed